processor/cumulativetodeltaprocessor/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/processor/cumulativetodeltaprocessor
+
+go 1.16
+
+require (
+	github.com/stretchr/testify v1.7.0
+	go.opentelemetry.io/collector v0.28.0
```

Comment:
This should match the version on the main branch

Reply:
Rebased with main branch.